src/ray/core_worker/core_worker.cc

```diff
@@ -339,7 +460,7 @@ CoreWorker::CoreWorker(const CoreWorkerOptions &options, const WorkerID &worker_
         << "max_pending_lease_requests_per_scheduling_category can't be 0";
     lease_request_rate_limiter_ =
         std::make_shared<ClusterSizeBasedLeaseRequestRateLimiter>(
-            /*kMinConcurrentLeaseCap*/ 10);
```

Comment:
google C++ coding style suggests to use `k`-prefixed CamelCase only on compile-time constant variables (i.e. constexpr).